pandas/core/indexes/datetimes.py

```diff
@@ -78,21 +78,26 @@ def _new_DatetimeIndex(cls, d):
 )
 class DatetimeIndex(DatetimeTimedeltaMixin):
     """
-    Immutable ndarray of datetime64 data, represented internally as int64, and
-    which can be boxed to Timestamp objects that are subclasses of datetime and
-    carry metadata such as frequency information.
+    Immutable ndarray of datetime64 data.
+
+    Represented internally as int64, and which can be boxed to Timestamp objects
+    that are subclasses of datetime and carry metadata such as frequency information.
 
     Parameters
     ----------
     data : array-like (1-dimensional), optional
         Optional datetime-like data to construct index with.
-    copy : bool
-        Make a copy of input ndarray.
     freq : str or pandas offset object, optional
         One of pandas date offset strings or corresponding objects. The string
         'infer' can be passed in order to set the frequency of the index as the
         inferred frequency upon creation.
     tz : pytz.timezone or dateutil.tz.tzfile
+        Timezone of the datetime.
+    normalize : bool, default False
+        Normalize start/end dates to midnight before generating date range.
+    closed : {None, 'left', 'right'}, optional
+        Make the interval closed with respect to the given frequency to
```

Comment:
I *think* that you can remove the ```None``` here, since we are saying that this parameter is optional.